src/lib/styles/global/layout.scss

```diff
@@ -32,3 +44,10 @@ header {
 
   --toolbar-pointer-events: all;
 }
+
+footer {
```

Comment:
to be honest, I don't think global styles is the way to go for a components library. Shouldn't the library just expose components and maybe a theme that the components use?

Not for this PR though...

Reply:
it's like for the [grid](https://ipi2f-uqaaa-aaaad-aabza-cai.ic0.app/components/grid) or the button - when it does not need JavaScript, I think it's more performant to not use JavaScript. Being said, for sure it would need design tokens as they are currently doing in II, that would be way cleaner! Also docs improvements.

Reply:
but you are right, there is a bit a mix though, probably worth a big clean up